bin/ch_core.c

```diff
@@ -330,7 +333,7 @@ void run_user_command(char *argv[], char *initial_dir)
 
 /* Wait for semaphore sem for up to timeout seconds. If timeout or an error,
    exit unsuccessfully. */
-void sem_timedwait_relative(sem_t *sem, int timeout)
+void sem_timedwait_relative(sem_t * const sem, int const timeout)
```

Comment:
timeout should not be const. I'm of the opinion that only pointers should be const.